server/helpers/filters.js

```diff
@@ -12,6 +12,7 @@ export const FILTER_TYPE = {
   INTEGER: 'integer',
   OBJECT_ID: 'objectId',
   STRING: 'string',
+  EXACT_STRING: 'exact string',
```

Comment:
Judging by how we named `objectId`. Also, this list is arranged in alphabetical order.

```suggestion
  EXACT_STRING: 'exactString',
```